lib/foreman_maintain/report.rb

```diff
@@ -22,6 +24,17 @@ def sql_setting(name)
       (result || {})['value']
     end
 
+    def flatten(hash, prefix = '')
+      hash.each_with_object({}) do |(key, value), result|
+        new_key = "#{prefix}#{prefix.empty? ? '' : ':'}#{key}"
```

Comment:
Using a colon as a separator is a bit unfortunate, considering keys might contain double colons
```
facts_by_type:ForemanAnsible::FactName:type: ForemanAnsible::FactName
facts_by_type:ForemanAnsible::FactName:min_update_time: '2024-10-15 10:37:23.592163'
facts_by_type:ForemanAnsible::FactName:max_update_time: '2024-12-02 09:06:55.684512'
facts_by_type:ForemanAnsible::FactName:count: '2679'
```